tests/x11regressions/x11regressions_setup.pm

```diff
@@ -14,19 +14,14 @@ use testapi;
 sub run() {
     my $self = shift;
 
-    x11_start_program("xterm");
-
-    # grant permission for default user to access serial port
-    type_string "xdg-su -c 'chown $username /dev/$serialdev'\n";
-    wait_still_screen;
+    # cmd1 grant user permission to access serial port until next reboot
+    # cmd2 get permanent user permission to access serial port even if reboot
+    my $cmd1 = "chown $username /dev/$serialdev";
+    my $cmd2 = "gpasswd -a $username dialout";
 
-    if ($password) {
-        type_password;
-        send_key "ret";
-    }
+    x11_start_program("xterm");
 
-    wait_still_screen;
-    save_screenshot;
+    script_sudo "$cmd1; $cmd2";
```

Comment:
what I thought is that the script_sudo was called here acturally came from lib/susedistribution.pm, and in this script_sudo, it defined by using su -c, not sudo:
type_string "clear; su -c \'$prog\'\n";

log here: 
http://147.2.207.208/tests/911/file/autoinst-log.txt

that was why i combined the cmds into 1 line.
anyway I will split it into 2 lines.